Update schedule.yml

debpaul opened the pull request:
Edited the Guest Speaker List and Titles.
@jlpereira is it possible that for this type of section, we could also have a **description** for each speaker/title? (So we could describe what each is about? I realize the that the **description** option exists for the other YML types (unconference / symposium). Let me know if this just not possible or desirable. Thanks.

debpaul committed:
Update schedule.yml

jlpereira commented:
@debpaul it should be possible, I've just made a change to allow schedules without days and time, so you should be able to set the type for it

debpaul committed:
Update schedule.yml

fixed name typo

debpaul committed:
Update schedule.yml

added some descriptions. @mjy please edit. @jlpereira please check my YML formatting of the descriptions, thanks!

jlpereira commented:
@debpaul break lines and check the indentation.,

For example:

```yml
      - speakers:
          - Dan Katz
        title: The Role of Research Software Engineers (RSE) and (long-term) open source science.
        description:
          'At this Research Software Engineering Round Table join us to discover just what is an RSE? We expect a lively discussion touching on evolving RSE communities.
          One key topic will be open source considerations for thinking about software architecture and software (re)use. For example, if we aim to produce software components that could be shared and reused,
          we need to think about a) what can effectively be shared? b) how do others discover these re-useable pieces exist? c) what architecture best supports the long-term existence/usability of the software?'
 ```

debpaul commented:
> @debpaul break lines and check the indentation.,
> 
> For example:
> 
> ```yaml
>       - speakers:
>           - Dan Katz
>         title: The Role of Research Software Engineers (RSE) and (long-term) open source science.
>         description:
>           'At this Research Software Engineering Round Table join us to discover just what is an RSE? We expect a lively discussion touching on evolving RSE communities.
>           One key topic will be open source considerations for thinking about software architecture and software (re)use. For example, if we aim to produce software components that could be shared and reused,
>           we need to think about a) what can effectively be shared? b) how do others discover these re-useable pieces exist? c) what architecture best supports the long-term existence/usability of the software?'
> ```

Hi @jlpereira thanks! I'll edit and then you can have a look see if I've understood what to do.

debpaul committed:
Update schedule.yml

edited YML per @jlpereira guidance.

jlpereira commented:
It's perefct now @debpaul

debpaul commented:
@mjy I'm going to merge this pull request now. I know you have not edited yet. There are some key bits that needed to change and I'm guessing you can edit other text when you add blurbs for the other guest events. Thanks @jlpereira for your help.
